{{ cookiecutter.package_name }}/.github/workflows/ci.yml

```diff
@@ -73,23 +73,6 @@ jobs:
       envs: |
         - linux: build_docs
 
-  cron:
-    if: |
-      github.event_name == 'workflow_dispatch' || (
-        github.event_name == 'pull_request' &&
-        contains(github.event.pull_request.labels.*.name, 'Run cron CI')
-      )
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
-    with:
-      default_python: '{{ default_python }}'
-      submodules: false
-      coverage: codecov
-      toxdeps: tox-pypi-filter
-      envs: |
-        - linux: py311-devdeps
```

Comment:
This job still needs to be run.

Reply:
where do you think I should put it instead? under test?

Reply:
I got no good idea, I think that's the only place it can go.

Reply:
yeah same, will add now